### ISSUE TITLE
Parameter "value" in trackStructuredEvent is not truly optional

### ISSUE THREAD

**Issue by mattmikolay** [snowplow-incubator/snowplow-react-native-tracker#25]
According to the [documentation for the `trackStructuredEvent` method](https://github.com/snowplow-incubator/snowplow-react-native-tracker#track-a-structured-event), the numeric `value` parameter is optional.

However, even after upgrading to the latest alpha (v0.1.0-alpha.13), I receive the following error when `undefined` or `null` are passed into `trackStructuredEvent` as the `value` argument:

```
Argument 4 (double) of RNSnowplowTracker.trackStructuredEvent must not be null
```

I see in a4bba22 that `label` and `property` were made nullable. I can confirm that I am successfully passing `null` as the `property` argument, but it appears there's still an issue with `value`.

**Comment by TuurDutoit** [comment]
We ran into the same problem at DataCamp. We're on 0.1.0-alpha.16

**Comment by colmsnowplow** [comment]
I thought I had posted a reply to this a long time ago :s apologies for the silence. 

As far as I can remember this a remnant of a workaround that is no longer needed.

This should get resolved in the next release.

**Comment by colmsnowplow** [comment]
Update on this! should be fixed in the just-pushed 0.1.0 release. Please do reopen or open a new issue if you discover any remaining unexpected behaviour.
